plugins/builds/get.js

```diff
@@ -23,27 +23,28 @@ module.exports = () => ({
             }
         },
         handler: (request, reply) => {
-            const factory = request.server.app.buildFactory;
+            const buildFactory = request.server.app.buildFactory;
 
-            return factory.get(request.params.id)
-                .then((model) => {
-                    if (!model) {
+            return buildFactory.get(request.params.id)
+                .then((buildModel) => {
+                    if (!buildModel) {
                         throw boom.notFound('Build does not exist');
                     }
 
-                    if (Array.isArray(model.environment)) {
-                        return reply(model.toJson());
+                    if (Array.isArray(buildModel.environment)) {
+                        return reply(buildModel.toJsonWithSteps());
```

Comment:
How about another `build.toJson()` cases like this line? Do we need adding steps same with here after removing `builds.steps`?
https://github.com/screwdriver-cd/screwdriver/blob/b9045682bcc43af1846a9cd9d15400fd79c5bdc5/plugins/builds/update.js#L184

Reply:
I fixed.